exist-core/src/main/java/org/exist/util/io/ContentFile.java

```diff
@@ -42,4 +43,7 @@ default InputStream newInputStream() throws IOException {
        return new ByteArrayInputStream(getBytes());
     }
 
+    default OutputStream newOutputStream() throws IOException {
```

Comment:
Whilst in this area, could you add the missing Javadoc to the interface class and methods please?

Reply:
@adamretter I will do that according to the comment of @dizzzz in the next moments...